packages/subgraph/subgraph.yaml

```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.5
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: PoolMetadataRegistry
+    network: goerli
+    source:
+      address: "0xEBfADf723E077c80f6058dC9c9202bB613DE07cf"
```

Comment:
I'm not knowledgeable in subgraph, but I feel this file should be automatically generated, no?

Reply:
On Balancer's repo there's a script generate-manifests to produce subgraph.yaml from a template. I made the same here and deleted this. 😉